pypeit/tests/test_runpypeit.py

```diff
@@ -139,7 +139,7 @@ def test_run_pypeit():
     specObjs = specobjs.SpecObjs.from_fitsfile(spec1d_file)
 
     # Flexure
-    assert specObjs[0].FLEX_SHIFT_TOTAL != 0., 'Flexure not calculated!'
+    assert abs(specObjs[0].FLEX_SHIFT_TOTAL + 0.7792376147143614) < 0.1
```

Comment:
Is the number too specific?  I.e., could you just replace it with 0.78?  And/or can you use `np.isclose`?

Reply:
Agreed - I've updated the number, and changed to `np.isclose` - thanks! I'll merge when travis passes

Reply:
If you use `isclose`, you'll need to loosen the tolerance.
I do think that is a good idea.

Reply:
OK - I've updated the code and the tests pass (I was doing two stupid things 😊)